src/panoramic/cli/cli.py

```diff
@@ -176,3 +195,16 @@ def detect_joins(target_dataset: str, yes: bool, diff: bool, overwrite: bool):
 
     if not detect_joins_command(target_dataset=target_dataset, diff=diff, overwrite=overwrite, yes=yes):
         sys.exit(1)
+
+
+@cli.command(context_settings=dict(ignore_unknown_options=True), help='Run dbt command', cls=DbtCommand)
+@click.argument('dbt_args', nargs=-1, type=click.UNPROCESSED)
+def dbt(dbt_args: Tuple[str]):
```

Comment:
This could use some test (unit and ideally some `e2e` too) - we could add it in a followup, up to you.

Reply:
Yeah was the e2e cause it seems like we'd be testing DBT itself but I'll look into it